twenty-first/src/shared_math/traits.rs

```diff
@@ -74,18 +75,19 @@ pub trait New {
 pub trait FromVecu8 {
     #[must_use]
     #[allow(clippy::wrong_self_convention)]
-    fn from_vecu8(&self, bytes: Vec<u8>) -> Self;
+    fn from_vecu8(bytes: Vec<u8>) -> Self;
 }
 
-pub trait PrimeField:
+pub trait FiniteField:
     Clone
     + Eq
     + Display
     + Serialize
     + DeserializeOwned
     + PartialEq
     + Debug
-    + IdentityValues
```

Comment:
It seems that this trait can now be removed, too?

Reply:
That's weird, I did not find this line in my version. 7603b93b does delete the (unimplemented) trait though.